bciers/apps/registration/vitest.config.mts

```diff
@@ -24,7 +24,7 @@ export default defineConfig({
       provider: "v8",
     },
     setupFiles: [
-      "../../libs/shared/testConfig/src/global.tsx",
+      "../../libs//testConfig/src/global.tsx",
```

Comment:
There's an extra slash after `libs` in a few places

Reply:
Haha thanks for catching these, it was more than just a few. I must have messed up my find and replace 😅